packages/manager/modules/pci/src/projects/new/payment/components/challenge/challenge.class.js

```diff
@@ -0,0 +1,43 @@
+import get from 'lodash/get';
+
+import {
+  CHALLENGER_CREDIT_CARD_LENGTH,
+  CHALLENGER_ERROR_CODES,
+} from './constants';
+
+export default class PciPaymentMethodChallenge {
+  constructor(options = {}, ibanValidator) {
```

Comment:
`ibanValidator` is not an injected dependency ? No need to `/* @ngInject */` ?

Reply:
Nope. Here I construct the object with the `ibanValidator` from `ovhPaymentMethodHelper`. So no need to have an injection here ;-)

Reply:
Ok :)